common/protob/messages-cardano.proto

```diff
@@ -104,7 +104,6 @@ message CardanoSignTx {
         repeated uint32 address_n = 1;  // BIP-32 path to derive the key from master node
```

Comment:
I am not sure about the name of this field. It is not an address anymore in Shelley, it is just path to derive the public spending key used for signing. But it should not be resolved in this PR anyway.